cvat/apps/dataset_manager/views.py

```diff
@@ -133,41 +154,52 @@ def export(dst_format, project_id=None, task_id=None, job_id=None, server_url=No
-            if not osp.exists(output_path):
-                with tempfile.TemporaryDirectory(dir=cache_dir) as temp_dir:
-                    temp_file = osp.join(temp_dir, 'result')
-                    export_fn(db_instance.id, temp_file, dst_format,
-                        server_url=server_url, save_images=save_images)
-                    os.replace(temp_file, output_path)
-
-                scheduler: Scheduler = django_rq.get_scheduler(
-                    settings.CVAT_QUEUES.EXPORT_DATA.value
-                )
-                cleaning_job = scheduler.enqueue_in(
-                    time_delta=cache_ttl,
-                    func=clear_export_cache,
-                    file_path=output_path,
-                    file_ctime=instance_update_time.timestamp(),
-                    logger=logger
-                )
-                logger.info(
-                    "The {} '{}' is exported as '{}' at '{}' "
-                    "and available for downloading for the next {}. "
-                    "Export cache cleaning job is enqueued, id '{}'".format(
-                        db_instance.__class__.__name__.lower(),
-                        db_instance.name if isinstance(
-                            db_instance, (Project, Task)
-                        ) else db_instance.id,
-                        dst_format, output_path, cache_ttl,
-                        cleaning_job.id
-                    )
-                )
+            if osp_exists(output_path):
+                # Update last modification time to prolong the export lifetime
+                # and postpone the file deleting by the cleaning job
+                os.utime(output_path, None)
+                return output_path
+
+        with tempfile.TemporaryDirectory(dir=cache_dir) as temp_dir:
+            temp_file = osp.join(temp_dir, 'result')
+            export_fn(db_instance.id, temp_file, dst_format,
+                server_url=server_url, save_images=save_images)
+            with get_export_cache_lock(
+                output_path,
+                ttl=EXPORT_LOCK_TTL,
+                acquire_timeout=EXPORT_CACHE_LOCK_ACQUISITION_TIMEOUT,
+            ):
+                os.replace(temp_file, output_path)
+
+        scheduler: Scheduler = django_rq.get_scheduler(settings.CVAT_QUEUES.EXPORT_DATA.value)
+        cleaning_job = scheduler.enqueue_in(
+            time_delta=cache_ttl,
+            func=clear_export_cache,
+            file_path=output_path,
+            file_ctime=instance_update_time.timestamp(),
+            logger=logger,
+        )
+        _patch_scheduled_job_status(cleaning_job)
+        logger.info(
+            "The {} '{}' is exported as '{}' at '{}' "
+            "and available for downloading for the next {}. "
+            "Export cache cleaning job is enqueued, id '{}'".format(
+                db_instance.__class__.__name__.lower(),
+                db_instance.name if isinstance(db_instance, (Project, Task)) else db_instance.id,
```

Comment:
```suggestion
                db_instance.id,
```
I think we don't really need task names in logs.

Reply:
Agree, updated